lib/manager/travis/extract.ts

```diff
@@ -22,6 +24,39 @@ export function extractPackageFile(content: string): PackageFile | null {
       currentValue: currentValue.toString(),
     }));
   }
+
+  // Handle the matrix syntax
+  let matrix_include: TravisMatrixItem[] | null;
```

Comment:
```suggestion
  let matrix_include: TravisMatrixItem[] | undefined;
```

if you don't assign `null` it's undefined :wink: